packages/seacas/libraries/ioss/src/exodus/Ioex_DatabaseIO.C

```diff
@@ -171,6 +171,9 @@ namespace Ioex {
         IOSS_ERROR(errmsg);
       }
     }
+
+    open_root_group();
```

Comment:
These probably need to be the `open_root_group_nl()` and `open_child_group_nl(0)` since I think we are already in a mutex lock state here.